lib/shared/store.js

```diff
@@ -308,7 +318,9 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         })
       }
 
-      return state.setIn([ 'selection', 'drive' ], Immutable.fromJS(action.data))
+      const stateDrives = state.getIn([ 'selection', 'drives' ])
```

Comment:
Perhaps this variable should be renamed to `selectionDrives` for consistency with line 457?